oggm/utils/_downloads.py

```diff
@@ -741,6 +742,43 @@ def _download_arcticdem_from_cluster_unlocked():
     return outpath
 
 
+def _download_aw3d30_file(zone):
+    with _get_download_lock():
+        return _download_aw3d30_file_unlocked(zone)
+
+
+def _download_aw3d30_file_unlocked(zone):
+    """Checks if the AW3D30 data is in the directory and if not, download it.
+    """
+
+    # extract directory
+    tmpdir = cfg.PATHS['tmp_dir']
+    mkdir(tmpdir)
+    outpath = os.path.join(tmpdir, zone + '_AVE_DSM.tif')
+
+    # check if extracted file exists already
+    if os.path.exists(outpath):
+        return outpath
+
+    # Did we download it yet?
+    ftpfile = ('ftp://ftp.eorc.jaxa.jp/pub/ALOS/ext1/AW3D30/release_v1804/'
+               + zone + '.tar.gz')
+    dest_file = file_downloader(ftpfile)
```

Comment:
this doesn't work right now

Reply:
requests does not support downloading from ftp. I searched quite a bit on how to do it, and only found ugly hacks and conclusions that it's an old protocol and they want people to use https instead.

Reply:
This is a clear incentive to solve the license issue. Since we have the files on cluster we could either:
- make them open and download from there
- hide them behind a password and use the same technique as for Tandem-X with a local credential file. Then we can control who gets the password (my preferred approach from now)